sympy/functions/special/tests/test_bessel.py

```diff
@@ -338,7 +338,7 @@ def test_bessel_nan():
 
 def test_conjugate():
     from sympy import conjugate, I, Symbol
-    n, z, x = Symbol('n'), Symbol('z', real=False), Symbol('x', real=True)
+    n, z, x = Symbol('n'), Symbol('z', extended_real=False), Symbol('x', real=True)
```

Comment:
I believe it should be `n, z, x = Symbol('n'), Symbol('z', extended_real=False), Symbol('x', extended_real=True)`

Reply:
In this PR BesselBase._eval_conjugate is
```
    def _eval_conjugate(self):
        z = self.argument
        if z.is_extended_negative is False:
            return self.__class__(self.order.conjugate(), z.conjugate())
```
I changed the test to exclude the possibility of z being -oo but now I wonder if I `_eval_conjugate` should be changed instead...

Reply:
Actually I think that `_eval_conjugate` is fine. I've changed it to `Symbol('x', extended_real=True)` as suggested.